### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by robbiehammond**
Code currently in the server, in it's (im)perfect form.

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -1,82 +1,136 @@
+
```
Comment (edf42001) [inline client/index.html]: Seriously what's going on with this file? Like how does this even happen? Anyway please fix

Comment (edf42001) [reply client/index.html]: Are they windows line endings? Or is it actually like that?

**Review by edf42001 — CHANGES_REQUESTED** [review]
Good start, just fix index.html

File: client/index.html (hunk 1 above)
Comment (robbiehammond) [reply client/index.html]: It might just be vim formatting my pasting weird, I’ll fix

**Review by edf42001 — APPROVED** [review]
nice fix

**Pull request merged**